src/main/java/gov/usgs/owi/nldi/controllers/CharacteristicsController.java

```diff
@@ -75,4 +75,18 @@ public void getCharacteristicData(HttpServletRequest request, HttpServletRespons
 		}
 		logService.logRequestComplete(logId, response.getStatus());
 	}
+
+	@GetMapping(value="{featureSource}/{featureID}/basin")
+	public void getBasin(HttpServletRequest request, HttpServletResponse response,
+			@PathVariable(LookupDao.FEATURE_SOURCE) String featureSource,
+			@PathVariable(Parameters.FEATURE_ID) String featureID) throws IOException {
+		BigInteger logId = logService.logRequest(request);
+		try {
+			streamBasin(response, getComid(featureSource, featureID));
+		} catch (Throwable e) {
```

Comment:
I've never seen Throwable used in a catch statement before. Could you elaborate on why Throwable is appropriate here vs. Exception? I understand that Throwable is a parent of Exception and Error, and that you want to catch a broad suite of abnormal conditions with this block. How important is it to catch Errors as well as Exceptions? Is this an appropriate place to handle an OutOfMemoryError or similar Error subclass?